pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written (sometime in 1999) to be a drop-in replacement for the (previously) non-free mpg123 player.
+> More information: <http://mpg321.sourceforge.net/>.
+
+- Play an audio source exactly N=3 times:
+
+`mpg321 -l {{3}} {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play an audio source forever (0 means until interrupted):
+
+`mpg321 -l 0 {{path/to/file_a|URL}} {{path/to/file_b|URL}} {{...}}`
+
+- Play a whole folder (recursively):
+
+`mpg321 -B {{path/to/directory}}`
+
+- Enable *Basic Keys* ( `*` or `/` - Increase or decrease volume, `n` - Skip song, `m` - Mute/unmute.) while playing:
```

Comment:
Markdown in descriptions isn't really supported apart from backticks. I would suggest to remove the asterisks.

Reply:
ok, done!